api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java

```diff
@@ -428,19 +428,21 @@ public QueryJobConfiguration previewBigQueryJobConfig(DataSetPreviewRequest requ
 
     // If pre packaged all survey concept set is selected create a temp concept set with concept ids
     // of all survey questions
-    if (CONCEPT_SETS_NEEDING_PREPACKAGED_SURVEY.contains(prePackagedConceptSet)) {
+    if (prePackagedConceptSet.contains(SURVEY)
+        || prePackagedConceptSet.contains(PrePackagedConceptSetEnum.BOTH)) {
```

Comment:
As of this PR i am still keeping it just for backward compatibility, For all new Dataset/update BOTH will be translated to SURVEY + PERSON rather than both